pyproject.toml

```diff
@@ -17,7 +17,7 @@ dbt-diagrams = "dbt_diagrams.cli:cli"
 python = ">=3.9,<3.13"
 click = "8.1.3"
 pyyaml = ">=6.0"
-pydantic = "1.10.13"
+pydantic = "^2.11.5"
```

Comment:
@blemmers Why a specific version of `pydantic`? Can we support something like `>=2,<3`?